commands/zrange.md

```diff
@@ -51,9 +51,13 @@ ZRANGE zset (5 (10 BYSCORE
 
 Will return all the elements with `5 < score < 10` (5 and 10 excluded).
 
-## Rev ranges
+## Reverse ranges
 
-Using the `REV` option reverses the sorted set, with index 0 as the element with the highest score. In this case, `<start>` must be less than or equal to `<stop>` to return anything. However, if the `BYSCORE` option is selected, the `<start>` is the highest score to consider, and `<stop>` is the lowest score to consider, therefore `<start>` must be less than or equal to `<stop>`
+Using the `REV` option reverses the sorted set, with index 0 as the element with the highest score.
+
+By default, `<start>` must be less than or equal to `<stop>` to return anything.
+
+However, if the `BYSCORE`, or `BYLEX` options are selected, the `<start>` is the highest score to consider, and `<stop>` is the lowest score to consider, therefore `<start>` must be greater than or equal to `<stop>` in order to return anything.
```

Comment:
No need to add a blank line between them. A blank line causes a new paragraph. A single linebreak does not produce a linebreak in the rendered HTML, but it's only for a smaller diff in future changes.

At least the second and third sentences should be in the same paragraph, I think.